src/libraries/Common/tests/TestUtilities/System/Buffers/BoundedMemory.Creation.cs

```diff
@@ -82,10 +87,15 @@ private static void FillRandom(Span<byte> buffer)
             {
                 return AllocateWithoutDataPopulationWindows<T>(elementCount, placement);
             }
-            else
+            else if (!IsMonoRuntime && (RuntimeInformation.IsOSPlatform(OSPlatform.FreeBSD) || RuntimeInformation.IsOSPlatform(OSPlatform.OSX) || RuntimeInformation.IsOSPlatform(OSPlatform.Linux)))
```

Comment:
is `IsOSPlatform` really needed here?

Reply:
Hmm, it might not be? I hadded it there to prevent wasm from calling into it, but now that I have `IsMonoRuntime`, it may not be needed.

Reply:
you can use `OperatingSystem.IsBrowser()` then

Reply:
That would be a lot better use, thanks